worker/caasapplicationprovisioner/application.go

```diff
@@ -257,7 +257,18 @@ func (a *appWorker) loop() error {
 		if done {
```

Comment:
Maybe add a timer to trip this every 10 seconds or so. I don't think we can do a watcher to automatically do this.